lib/net/rpc.go

```diff
@@ -0,0 +1,38 @@
+package net
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	gonet "net"
+	"net/http"
+	"net/rpc"
+)
+
+func dialHTTPPath(dialer Dialer, network, address, path string) (
+	*rpc.Client, error) {
+	var err error
```

Comment:
I thought about that, but figured there is value in letting this DialHTTPPath work just like the one in golang which does not provide for passing an empty string for path.  Besides, it looks like you already change an empty path to rpc.DefaultRPCPath in rpcclientpool package. So it doesn't seem right to have that logic twice.